docs(getstarted): add Beyla for Docker (#4051)
kubeshop/tracetest: docs/docs/getting-started/configure-trace-ingestion.mdx

[Visit the example in GitHub, here.](https://github.com/kubeshop/tracetest/tree/main/examples/getting-started/ruby)
:::

</TabItem>
<TabItem value="docker" label="Docker">

> You can configure [Grafana Beyla](https://grafana.com/oss/beyla-ebpf/) to autoinstrument Docker services and export traces to Tracetest.

Reviewer: ```suggestion
You can configure [Grafana Beyla](https://grafana.com/oss/beyla-ebpf/) to auto-instrument Docker services and export traces to Tracetest.
```

> 1. Add Beyla to your `docker-compose.yaml`

Reviewer: ```suggestion
1. Add Beyla to your `docker-compose.yaml`:
```

```yaml title="docker-compose.yaml"
services:
  autoinstrument:
    image: grafana/beyla:latest
    privileged: true
    environment:
      OTEL_EXPORTER_OTLP_ENDPOINT: "http://<tracetest-agent>:4317"
      # OTEL_EXPORTER_OTLP_HEADERS: "x-tracetest-token:<token>"
      BEYLA_OPEN_PORT: "3000,8080" # Beyla will autoinstrument services on these ports
      BEYLA_TRACE_PRINTER: "text"

# ...
```

> 2. Add `pid: service:autoinstrument` to services in `docker-compose.yaml`

Reviewer: ```suggestion
2. Add `pid: service:autoinstrument` to services in `docker-compose.yaml`:
```
